src/Uno.UI.RuntimeTests/Tests/Windows_Globalization/Given_ApplicationLanguages.cs

```diff
@@ -12,17 +13,20 @@ public class Given_ApplicationLanguages
 		[TestCleanup]
 		public void CleanUp()
 		{
-			ApplicationLanguages.PrimaryLanguageOverride = null;
+			ApplicationLanguages.PrimaryLanguageOverride = string.Empty;
 		}
 
 		[TestMethod]
 		public void Test_Chinese_With_Script_Subtag()
 		{
+			
 			ApplicationLanguages.PrimaryLanguageOverride = "zh-Hans-CN";
 
 			ApplicationLanguages.Languages.First().Should().Be("zh-Hans-CN");
 			CultureInfo.CurrentCulture.Name.Should().BeOneOf("zh-CN", "zh-Hans-CN");
 			CultureInfo.CurrentUICulture.Name.Should().BeOneOf("zh-CN", "zh-Hans-CN");
+
+			ApplicationData.Current.LocalSettings.Values["Uno.PrimaryLanguageOverride"].Should().Be("ch-Hans-CN");
```

Comment:
I'm not sure this is the appropriate kind of test.

Reply:
It's definitely not "black box testing", but in this case, I think it's acceptable since we won't restart the app just for that.

But a another test would be to set the local settings and find a way to force the ApplicationLanguages to rerun its initialization.